openedx/core/djangoapps/programs/migrations/0001_initial.py

```diff
@@ -23,6 +23,7 @@ class Migration(migrations.Migration):
                 ('public_service_url', models.URLField(verbose_name='Public Service URL')),
                 ('api_version_number', models.IntegerField(verbose_name='API Version')),
                 ('enable_student_dashboard', models.NullBooleanField(verbose_name='Enable Student Dashboard Displays')),
+                ('cache_ttl', models.PositiveIntegerField(default=0, help_text='Specified in seconds. Enable caching by setting this to a value greater than 0.', verbose_name='Cache Time To Live')),
```

Comment:
We can't change the initial migrations any more.  This will have to be a new migration to add this field.